lib/portlayer/network/config.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package network
+
+import (
+	"net"
+
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/lib/metadata"
+)
+
+var Config Configuration
+
+type Configuration struct {
+	// The default bridge network supplied for the Virtual Container Host
+	BridgeNetwork string `vic:"0.1" scope:"read-only" key:"bridge_network"`
+	// Published networks available for containers to join, keyed by consumption name
+	ContainerNetworks map[string]*ContainerNetwork `vic:"0.1" scope:"read-only" key:"container_networks"`
+}
+
+type ContainerNetwork struct {
```

Comment:
Agreed, composition is preferred. This requires and addition to the extraConfig code of an `inline` annotation to suppress the extra key layer that would appear from reflection. I don't think it's in scope for this but have opened #1145 to track it.